tutorials/db-migration-spanner-handle-increasing-pks/index.md

```diff
@@ -0,0 +1,363 @@
+        sudo chown striim /opt/striim/lib/CustomFunctions.jar
+        sudo chmod +x /opt/striim/lib/CustomFunctions.jar
+
+5. Restart striim application
+
+        sudo systemctl stop striim-node
+        sudo systemctl stop striim-dbms
+        sudo systemctl start striim-dbms
+        sudo systemctl start striim-node
+
+### Create Initial load pipeline
+Initial load is meant to bulk load existing data. It reads data as on job start time and once finished it will quiesce (pause).
+Any further changes in data after job start time will not be read by this job.
+
+1. Save below code as file `mysql_to_spanner_initial_load.tql` on your local computer. You will import it into striim.
+
+        drop APPLICATION mysql_to_spanner_initial_load CASCADE;
+        
+        IMPORT STATIC CustomFunctions.*;
+        
+        CREATE APPLICATION mysql_to_spanner_initial_load;
+        
+        CREATE FLOW sample1;
+        
+        CREATE OR REPLACE SOURCE mysql_dbreader USING Global.DatabaseReader (
+          Username: 'root',
+          DatabaseProviderType: 'Default',
+          FetchSize: 10,
+          adapterName: 'DatabaseReader',
+          QuiesceOnILCompletion: true,
+          Password_encrypted: 'false',
+          ConnectionURL: 'jdbc:mysql://PRIMARY_IP:3306/employeedb',
+          Tables: 'employeedb.signin_log',
+          Password: 'password123',
+          ReturnDateTimeAs: 'String')
+        OUTPUT TO sample_raw;
+        
+        CREATE STREAM SampleModifiedData OF Global.WAEvent;
+        
+        CREATE CQ POPULATE_SAMPLE_TABLE
+        INSERT INTO SampleModifiedData
+        SELECT * from sample_raw
+        MODIFY(data[0] = bitReverseInt(data[0]));
+        
+        CREATE TARGET spanner_sample USING Global.SpannerWriter (
+          InstanceID: 'spanner-tgt',
+          ServiceAccountKey: '/opt/striim/striim-spanner-key.json',
+          BatchPolicy: 'EventCount: 100, Interval: 10s',
+          ParallelThreads: '',
+          Tables: 'employeedb.signin_log,employeedb.signin_log' )
+        INPUT FROM SampleModifiedData;
+        
+        END FLOW sample1;
+        
+        END APPLICATION mysql_to_spanner_initial_load;
+
+2. Login to striim web ui. If needed, visit [deployment manager page](https://console.cloud.google.com/dm/deployments)
+ access striim web ui link and credentials. 
+3. Click on Apps > + Add App (on top right).
+4. Click import existing app and choose the mysql_to_spanner_initial_load.tql created in step 1. And click import.
+5. In the workflow, replace Primary_IP with MySQL’s ip address. Click mysql_dbreader component and change the Primary_IP 
+in the right window showing connection url (screenshot below). And click save
+![initial load](4_striim_initial_load.png)
+
+6. Created > Deploy App > Deploy
+![initial load deploy](5_striim_deploy.png)
+
+7. Deploy > Start App. 
+8. You will soon observe rows flowing through striim and data being inserted inside the cloud spanner.
```

Comment:
Should we add a screenshot of how we can preview the data ? Also maybe another point on tracking progress ?
https://www.striim.com/docs/smsgc/en/striim-migration-service-user-guide/smsgc-workflow-overview/monitor-application-progress.html

```suggestion
8. You will soon observe rows flowing through striim and data being inserted inside the cloud spanner.
```